passport-config.js

```diff
@@ -24,8 +25,25 @@ function initialize(passport, getUserByEmail, getUserByID) {
         })
     }
     passport.use(new LocalStrategy({usernameField: 'email'}, authenticateUser));
+    passport.use(
+        new JWTstrategy(
+            {
+                secretOrKey: 'secretKey',
+                jwtFromRequest: req => req.cookies.jwt
```

Comment:
```suggestion
                const ExtractJwt = require("passport-jwt").ExtractJwt;
                jwtFromRequest:  ExtractJwt.fromAuthHeaderAsBearerToken();
```

Reply:
If the client-side is responsible for setting the the token as a cookie, shouldn't this be left the same?

Reply:
Our mobile app won't be sending cookie, all api request would be sending jwt in auth header as bearer token
All api requests would be sending the jwt token in the auth header

Reply:
web front end would extract jwt from cookie and attach it to the request header.

Reply:
Ah okay, I see, so the mobile client and admin client would need to send the jwt via the auth header.
And to logout, both clients would remove the auth header.

Reply:
logout would just be removing from cookie and localstorage. So the application won't have jwt token to attach to header with.